app/lib/package/backend.dart

```diff
@@ -480,31 +472,28 @@ class GCloudPackageRepository extends pub_server.PackageRepository {
   }
 
   /// Finishes the upload of a package.
-  @override
-  Future<pub_server.PackageVersion> finishAsyncUpload(Uri uri) async {
+  Future<PackageVersion> finishAsyncUpload(Uri uri) async {
```

Comment:
We could give this a better name... maybe `processUploadedBlob` or `publishUploadedBlob`